tests/collectors.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import yaml
+
+from .fixtures import plan_summary, plan_validator
+
+
+class FabricTestFile(pytest.File):
+
+  def collect(self):
+    """Read yaml test spec and yield test items for each test definition.
+
+    Test spec should contain a `module` key with the path of the
+    terraform module to test, relative to the root of the repository
+
+    All other top-level keys in the yaml are taken as test names, and
+    should have the following structure:
+
+    test-name:
```

Comment:
so something like

```yaml
module: foo
test-1:
  tfvars:
    - foo.tfvars
  inventory:
    - bar.yaml
```

Having `module` at the same level as the test keys triggers my OCD :) can we move test key/values under a `tests` key?  It would also free the namespace for stuff we might want to add later.

Reply:
I actually had the `tests` key before and removed it to simplify the file. However, I agree that we might want to keep the top-level namespace clean for future use.